### PR TITLE
dbeaver/dbeaver#23350 [CrateDB] Can't create table in data import

### LINKED ISSUE #23350: [CrateDB] Can't create table in data import
### Description

During import the DDL to create table would always have nullability argument in it wich leads to an error in CrateDB. The same bug is also present in generated DDL of any existing table.

### DBeaver Version

24.0.2

### Operating System

Any

### Database and driver

_No response_

### Steps to reproduce

1. Open import wizard
2. Select any CSV file
3. Set mapping to create
4. Start the import

**Expected:** Table is created
**Actual:** SQL error

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by serjiokov**
Fixed type of server for DDL generation

### CODE REVIEW

**Review by E1izabeth — APPROVED** [review]

File: plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/edit/PostgreTableColumnManager.java [hunk 1]
```diff
@@ -146,14 +149,20 @@ public DBSObjectCache<? extends DBSObject, PostgreTableColumn> getObjectsCache(P
 
     protected ColumnModifier[] getSupportedModifiers(PostgreTableColumn column, Map<String, Object> options)
     {
+        PostgreServerExtension serverType = column.getDataSource().getServerType();
+        if (serverType.getServerTypeName().equals(POSTGRE_SERVER_TYPE_CRATE_DB)) {
```
Comment (serge-rider) [inline plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/edit/PostgreTableColumnManager.java]: Let's add function to ServerType (e.g. supportsNotNull)

**Review by serge-rider — CHANGES_REQUESTED** [review]

File: plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/edit/PostgreTableColumnManager.java (hunk 1 above)
Comment (serjiokov) [reply plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/edit/PostgreTableColumnManager.java]: Done

**Review by LonwoLonwo — APPROVED** [review]

**Review by ShadelessFox — APPROVED** [review]

**Comment by uslss** [comment]
cannot import text fields due to an error - legacy driver was used. issue isn't reproducible against non-legacy driver.

https://github.com/dbeaver/dbeaver/assets/31996417/be54f8a6-fb48-438e-a2c1-7393ebc39bbe

[CUSTOMER_202404191520.csv](https://github.com/dbeaver/dbeaver/files/15457818/CUSTOMER_202404191520.csv)

**Comment by uslss** [comment]
verified

**Pull request merged**